src/makeCalculator.js

```diff
@@ -38,6 +38,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      operation(number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
```

Comment:
```suggestion
      this.result += number;
```

you need to use `this` instead of `calculator`

Reply:
Yeah, I tryed using `this `in callback methods `add, subtract `etc. But сontext `this `inside the callbacks equals undefined. I tryed `bind `context, but smth went wrong ((